softgroup/basis/common-router/common-router-impl/src/main/java/com/softgroup/common/router/impl/RootRouter.java

```diff
@@ -0,0 +1,18 @@
+package com.softgroup.common.router.impl;
+
+import com.softgroup.common.router.api.AbstractRouterHandler;
+import com.softgroup.common.router.api.CommonRouterHandler;
+import com.softgroup.common.router.api.factory.RouterFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by nikmlk on 02.03.17.
+ */
+
+@Component
+public class RootRouter<T extends CommonRouterHandler, F extends RouterFactory<T>> extends AbstractRouterHandler<T, F> {
```

Comment:
@melnyknikolay How do you suppose to create component from this? Where is generic types ?